src/kernel-loader/index.ts

```diff
@@ -252,8 +252,17 @@ async function initKernel() {
   })
 
   kernel.on('openUrl', ({ url }) => {
-    const newWindow = window.open(url, '_blank', 'noopener,noreferrer')
-    if (newWindow != null) newWindow.opener = null
+    try {
+      const u = new URL(url)
+      if (u.protocol === 'https:') {
+        const newWindow = window.open(u.toString(), '_blank', 'noopener,noreferrer')
+        if (newWindow != null) newWindow.opener = null
```

Comment:
this line may fail if the opened URL doesn't have the same domain

<img width="893" alt="Screen Shot 2022-08-25 at 22 41 23" src="https://user-images.githubusercontent.com/208789/186764990-cdde7c5f-f5db-4650-9d8d-ba9af3acb557.png">

Reply:
are we ok with this? Or do you prefer to change it?

Reply:
I don't understand, it shouldn't fail. It has an if `if (newWindow != null) newWindow.opener = null`

Reply:
should be `null` because of the `noopener` but if not and we try to overwrite `newWindow.opener`  it may fail

Reply:
shouldn't we add a `try/catch` and close it if we couldn't secure the interactions with the new window?

Reply:
Or remove the line entirely